src/WinRT.Runtime/Projections/INotifyCollectionChanged.net5.cs

```diff
@@ -15,26 +15,25 @@ namespace ABI.System.Collections.Specialized
 #endif
     static class INotifyCollectionChangedMethods
     {
-        private volatile static global::System.Runtime.CompilerServices.ConditionalWeakTable<object, NotifyCollectionChangedEventSource> _CollectionChanged;
-        private static global::System.Runtime.CompilerServices.ConditionalWeakTable<object, NotifyCollectionChangedEventSource> MakeCollectionChangedTable()
+        private volatile static global::System.Runtime.CompilerServices.ConditionalWeakTable<object, NotifyCollectionChangedEventHandlerEventSource> _CollectionChanged;
+        private static global::System.Runtime.CompilerServices.ConditionalWeakTable<object, NotifyCollectionChangedEventHandlerEventSource> MakeCollectionChangedTable()
         {
             global::System.Threading.Interlocked.CompareExchange(ref _CollectionChanged, new(), null);
             return _CollectionChanged;
         }
-        private static global::System.Runtime.CompilerServices.ConditionalWeakTable<object, NotifyCollectionChangedEventSource> CollectionChanged => _CollectionChanged ?? MakeCollectionChangedTable();
+        private static global::System.Runtime.CompilerServices.ConditionalWeakTable<object, NotifyCollectionChangedEventHandlerEventSource> CollectionChanged => _CollectionChanged ?? MakeCollectionChangedTable();
 
 
-        public static unsafe (Action<global::System.Collections.Specialized.NotifyCollectionChangedEventHandler>, Action<global::System.Collections.Specialized.NotifyCollectionChangedEventHandler>) Get_CollectionChanged(IObjectReference obj, object thisObj)
+        public static unsafe global::WinRT.Interop.EventSource<global::System.Collections.Specialized.NotifyCollectionChangedEventHandler> Get_CollectionChanged(IObjectReference obj, object thisObj)
```

Comment:
This is a new class, so this change is fine.  But looking at this, is there a breaking change here where these type of functions are now projected differently?

Reply:
Resolving this since it's the same as the one below.